src/index.ts

```diff
@@ -60,7 +60,7 @@ export class Pipeline implements Serializable {
     }
 
     add(...step: PotentialStep[]): this {
-        step.forEach(s => {
+        step.forEach((s) => {
```

Comment:
This is a rebase of #64

Reply:
I merged #64 just now, so this diff should go away.